<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatmapSettingsWidget</class>
 <widget class="QWidget" name="HeatmapSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_xAxis">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Название оси X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_xAxisLabel">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_yAxis">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Название оси Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_yAxisLabel">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="settingsTable">
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Отрисовка</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Интервал</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цвет</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
